### DISCUSSION TITLE
My domain "is already taken". What should I do?

### DISCUSSION THREAD

**Discussion by chupper100** [community/community#23556] [Other Feature Feedback, Questions, & Ideas]
<p>Related Information:</p>
<ul>
<li>I got the domain <code>alfie.cf</code> on Freenom (free domain).</li>
<li>Using Cloudflare as my DNS management.</li>
</ul>
<p>What happened?</p>
<ol>
<li>went to Cloudflare to set up that domain for some tests.</li>
<li>I visited <code>http://alfie.cf</code> and noticed that the domain is being HOSTED.</li>
<li>I took a random repository on GitHub Pages to check if it is eligible for Pages custom domain.</li>
<li>I got this error when I add <code>alfie.cf</code> as a custom domain:<br>
<code>The custom domain alfie.cf is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain.</code>
</li>
<li>Then, I followed <a href="https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages" class="inline-onebox" rel="noopener nofollow ugc">Verifying your custom domain for GitHub Pages - GitHub Docs</a> to verify my domain. It’s verified, but I got the same error when I tried to add <code>alfie.cf</code> as a custom domain.</li>
</ol>
<p>DNS records for <code>alfie.cf</code>:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/5/d533ff1d793034d0b47331253db7eb9bab4dcb81.png" data-download-href="/uploads/short-url/uq56p20CE37hojMbYQrr9v9J0mB.png?dl=1" title="Records" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/76856403/181077881-43a8d152-23ce-473f-a15b-432ea08b7e53.png" alt="Records" data-base62-sha1="uq56p20CE37hojMbYQrr9v9J0mB" width="690" height="190" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/5/d533ff1d793034d0b47331253db7eb9bab4dcb81_2_690x190.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/5/d533ff1d793034d0b47331253db7eb9bab4dcb81_2_1035x285.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/5/d533ff1d793034d0b47331253db7eb9bab4dcb81.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/5/d533ff1d793034d0b47331253db7eb9bab4dcb81_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Records</span><span class="informations">1138×315 31.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mishmanners** [comment, marked as answer]
<p>Check out this tutorial on enabling a custom domain on GitHub pages: <a href="https://dev.to/github/how-to-use-github-pages-to-host-your-website-even-with-multiple-repos-27k2" class="inline-onebox">How to use GitHub Pages to host your website, even with multiple repos - DEV Community</a></p>

**Comment by shd101wyy** [comment]
Hi @chupper100 , have you found a solution? I encountered the same issue. Thanks

**Reply by chupper100** [reply]
To be honest, it has been half a year, so I cannot recall the solution 😂. However, you should try to use CNAME and AAAA (or A) alternately.

**Reply by shd101wyy** [reply]
Thanks. I reached out to the GitHub support and they helped release my domain from another account 👍

**Reply by mishmanners** [reply]
Amazing! That's an awesome result. Glad our team could help 😄

**Reply by solyasa** [reply]
This is good experience for me

**Comment by ugfire** [comment]
am having the same issue in 2024. how do i reach the github support team? I only see community support
